MoEngagePluginBase/MoEngagePluginUtils.swift

```diff
@@ -28,6 +28,21 @@ public class MoEngagePluginUtils {
     }
     
     // MARK: InApp Utilities
+    static func inAppNavigationCampaignToJSON(inAppCampaign: MoEngageInAppCampaign, inAppAction: MoEngageInAppNavigationAction, identifier: String) -> [String: Any] {
+        let accountMeta = createAccountPayload(identifier: identifier)
+        
+        var inAppDataPayload = inAppCampaign.fetchInAppPaylaod()
+        
+        let actionPayload = inAppAction.fetchInAppNavigationActionPayload()
+        
+        inAppDataPayload[MoEngagePluginConstants.General.actionType] = MoEngagePluginConstants.General.navigation
```

Comment:
Should we hardcode actionType here ? Since MoEngageInAppNavigationAction is a sub class of MoEngageInAppAction, all values should be possible.

Reply:
@babulmoengage  done